packages/compass-generative-ai/src/store/atlas-ai-store.ts

```diff
@@ -1,12 +1,17 @@
-import { createStore, applyMiddleware } from 'redux';
+import { createStore, applyMiddleware, combineReducers } from 'redux';
 import thunk from 'redux-thunk';
-import reducer, {
+import signInReducer, {
   atlasServiceSignedOut,
   atlasServiceSignedIn,
-  atlasServiceTokenRefreshFailed,
+  //atlasServiceSignInTokenRefreshFailed,
+} from './atlas-signin-reducer';
+import optInReducer, {
+  atlasServiceSignInTokenRefreshFailed,
 } from './atlas-signin-reducer';
```

Comment:
How are we getting the `optInReducer` from the atlas-signin-reducer? Maybe an imports bug.